src/components/betting-chips/BettingChip.tsx

```diff
@@ -14,18 +14,27 @@ export type BettingChipProps = {
 
 export const BettingChip = observer((props: BettingChipProps) => {
   const { value, betOn } = props;
-  const { game } = useMainStore();
+  const { game, player } = useMainStore();
 
   const addToBet = () => {
     switch (betOn) {
       case "player":
-        game.addToPlayerBet(value);
+        if (value <= player.playerMoney) {
+          game.addToPlayerBet(value);
+          player.removePlayerMoney(value);
```

Comment:
Seems like `player.removePlayerMoney(value)` can be just moved after switch case statement